src/sentry/static/sentry/app/views/organizationGroupDetails/actions/index.tsx

```diff
@@ -251,7 +278,7 @@ class Actions extends React.Component<Props, State> {
             <Link
               className={buttonClassName}
               title={t('Open in Discover')}
-              to={this.getDiscoverUrl()}
+              to={disabled ? '' : this.getDiscoverUrl()}
```

Comment:
Why empty string? Would it make sense to disable it / not make it a link?

Reply:
as it has the` btn-button disabled` class it will look disabled in the UI. I did not want to change this component for now